src/System.Net.Requests/dir.props

```diff
@@ -1,8 +1,8 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.0.12.1</AssemblyVersion>
-    <PackageVersion>4.3.1</PackageVersion>
+    <AssemblyVersion>4.0.12.0</AssemblyVersion>
+    <PackageVersion>4.3.0</PackageVersion>
```

Comment:
This is the default so you don't need to specify it.

Is it possible to just do a `git revert 74916f6d0abdfd9cad65dd7f69b991680997ac3c`?